pymhf/core/utils.py

```diff
@@ -9,6 +13,30 @@
 #     with open(op.join(_internal.CWD, fname), "w") as f:
 #         f.write(json.dumps(res, indent=2))
 
+def is_window_launched(windowName):
+    return pwc.getWindowsWithTitle(windowName) is not None
+
+
+def get_window(windowName):
+    return pwc.getWindowsWithTitle(windowName)[0]
+
+
+def is_window_active(windowName):
+    return windowName == pwc.getActiveWindowTitle()
+
+
+def activate_window(windowName):
+    if is_window_launched(windowName):
+        get_window(windowName).activate()
+
+
+def set_nms_window_focus():
```

Comment:
this shouldn't be here. `pyMHF` is generic and not for NMS specifically. Any functions and methods need to be able to applied to any application, not just NMS

Reply:
fixed